### PR TITLE
Introduction fixes #91

### LINKED ISSUE #91: Introduction, including a brief problem description and end-user's requirements

### PR DESCRIPTION
**Pull request by eanker**

### CODE REVIEW

File: Documentation/General/finalReport/introduction.tex [hunk 1]
```diff
@@ -1 +1,18 @@
-\chapter{Introduction}
\ No newline at end of file
+\chapter{Introduction}
+
+The context project is a product in which bachelor students at the TU Delft get a task to develop a system in 10 weeks for a given problem.
+
+The main problem our context had to face was to design the music services of tomorrow.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/introduction.tex]: The task we were assigned / The question that was asked

File: Documentation/General/finalReport/introduction.tex [hunk 2]
```diff
@@ -1 +1,18 @@
...
+Our group had to design an application that would allow users to listen to music of a certain mood and chat with people in the same mood.\\
+
+The requirements of the system is that users can select a mood and a room and then join a room according to their mood.
+In the room they can get with each other, if they are logged in.
+A user can classify a song according to mood of the song.
+A user can also go the ranking game and classify unranked songs.
+The user gets points for classifying a songs.\\
+
+
+In this report we will explain MoodCat and how it is made.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/introduction.tex]: and how we have developed the system.

**Comment by jwgmeligmeyling** [comment]
:+1:

**Comment by TimvdLippe** [comment]
Changed some wording. Is good by me!

**Comment by jwgmeligmeyling** [comment]
Should be fine!

**Comment by GijsWeterings** [comment]
LGTM, merging!

**Pull request merged**